File: src/main/java/econtroller/controller/ControllerInit.java
package econtroller.controller;

import econtroller.ControllerConfiguration;
import se.sics.kompics.Init;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-11
 *
 */

public class ControllerInit extends Init {

	private ControllerConfiguration controllerConfiguration;

	public ControllerInit(ControllerConfiguration controllerConfiguration) {
		this.controllerConfiguration = controllerConfiguration;
	}

	public ControllerConfiguration getControllerConfiguration() {
		return controllerConfiguration;
	}

}

File: src/main/java/econtroller/controller/ControllerInit.scala
package econtroller.controller

import se.sics.kompics.Init
import econtroller.ControllerConfiguration

/**
 * @author Amir Moulavi
 * @date 2011-07-23
 *
 */
class ControllerInit(val controllerConfiguration:ControllerConfiguration) extends Init {
  def getControllerConfiguration = controllerConfiguration
}